HW5/task2.py

```diff
@@ -0,0 +1,64 @@
+# 2. Створити view яка на вхід приймає стиль музики. Має вивести місто
+# в якому найбільше слухають цей стиль музики. Якшо жанра немає вивести повідомлення.
+# genre - обов'язковий параметер.
+# /stats_by_city?genre=HipHop
+
+from flask import Flask
+from process_request import execute_query
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def Hello_world():
+    return "<p>Hello, world!</p>"
+
+
+@app.route('/stats_by_city')
+@use_kwargs(
+    {
+        'genre': fields.Str(
+            missing=None
+        )
+    },
+    location='query'
+)
+def find_city_genre(genre):
+    if genre:
+        query = """
+        --sql
+        SELECT 
+            customers.City, 
+            genres.Name, 
+            COUNT(*) AS GenreCounts
+        FROM 
+            customers
+        JOIN 
+            invoices ON customers.CustomerId = invoices.CustomerId
+        JOIN 
+            invoice_items ON invoices.InvoiceId = invoice_items.InvoiceId
+        JOIN 
+            tracks ON invoice_items.TrackId = tracks.TrackId
+        JOIN 
+            genres ON tracks.GenreId = genres.GenreId
+        WHERE 
+            genres.Name = ?
+        GROUP BY
+            customers.City
```

Comment:
fix this query to show the top city, also pay attention that few cities could have the biggest values of count
Here you need to use window function